src/components/axes/timeAxis.ts

```diff
@@ -280,12 +287,19 @@ export module Axis {
 
     protected _setup() {
       super._setup();
+      this._createDomElements();
+    }
+
+    private _createDomElements() {
+      d3.selectAll(".time-axis-tier").remove();
```

Comment:
This entire logic is a bit weird actually, because I was in doubts of how to do it properly. Can we maybe discuss this, so I see your thoughts?